TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/ObstacleCoursePath.java

```diff
@@ -14,8 +14,14 @@ public class ObstacleCoursePath extends LinearOpMode {
     public void runOpMode() {
         SampleMecanumDrive drive = new SampleMecanumDrive(hardwareMap);
 
-        // delete this
-        drive.setPoseEstimate(new Pose2d(5, 10, Math.toRadians(0)));
+        // first part
+        drive.setPoseEstimate(new Pose2d(-60, 60, Math.toRadians(270)));
+        Trajectory trajectoryGoForward = drive.trajectoryBuilder(new Pose2d(-60, 60, Math.toRadians(270)))
+                .splineTo(new Vector2d(-57.3, 29), Math.toRadians(270))
+                .build();
+        Trajectory trajectoryTurnLeft = drive.trajectoryBuilder(trajectoryFirstPart.end())
+                .splineTo(new Vector2d(5, 10), Math.toRadians(0))
+                .build();
 
```

Comment:
Be consistent with spaces between each part--either add empty line before each comment for the trajectory parts or delete empty lines